devscripts/update-version.py

```diff
@@ -50,5 +50,5 @@ def get_git_head():
 '''
 
 write_file('yt_dlp/version.py', VERSION_FILE)
-print(f'::set-output name=ytdlp_version::{VERSION}')
+write_file(os.getenv('GITHUB_OUTPUT'), f'ytdlp_version={VERSION}\n', 'a')
```

Comment:
if the env var doesnt exist (running locally), this should be skipped

Reply:
okay